ADMVEDU/Network/NetworkError.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  NetworkError.swift
+//  ADMVEDU
+//
+//  Created by Bobr, Andrey on 4.05.21.
+//
+
+import Foundation
+
+enum NetworkError: Error {
+    case notFound
+    case inavlidData
+    case unexpected(code: Int)
+
+}
+
+extension NetworkError {
+    var isFatal: Bool {
+        if case NetworkError.unexpected = self { return true }
+        else { return false }
+    }
+}
+
+extension NetworkError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .inavlidData:
+            return NSLocalizedString(
+                "The provided data is not valid.",
```

Comment:
1. this text should be moved to a localisation file (here and below)
2. let's incapsulate NSLocalizedString to 'var localized' of String ("localizationKey".localized), in this case comment can be nil